core/lib/zksync_core/src/lib.rs

```diff
@@ -550,11 +553,18 @@ pub async fn initialize_components(
         let eth_client =
             PKSigningClient::from_config(&eth_sender, &contracts_config, &eth_client_config);
         let nonce = eth_client.pending_nonce("eth_sender").await.unwrap();
+        let l1_batch_committer: Arc<dyn L1BatchCommitter> =
+            if std::env::var("VALIDIUM_MODE") == Ok("true".to_owned()) {
```

Comment:
probably will be fixed later, but just in case - we should have proper config for this, not access env var. For now of course just do however is better